### ISSUE TITLE
IDE(IDEA 2020.3 Ultimate Edition): Fatal Errors

### ISSUE THREAD

**Issue by Samarium150** [Him188/kotlin-jvm-blocking-bridge#5]
Partial stacktrace:
```
org.jetbrains.kotlin.idea.caches.resolve.KotlinIdeaResolutionException: Kotlin resolution encountered a problem while analyzing KtFile
    ...
Caused by: java.lang.NoSuchMethodError: 'org.jetbrains.kotlin.descriptors.ModuleCapability org.jetbrains.kotlin.analyzer.ModuleInfo$Companion.getCapability()'
    at net.mamoe.kjbb.ide.JvmBlockingBridgeIDEContainerContributor$registerModuleComponents$1.invoke(JvmBlockingBridgeIDEContainerContributor.kt:25)
    at net.mamoe.kjbb.ide.JvmBlockingBridgeIDEContainerContributor.registerModuleComponents(JvmBlockingBridgeIDEContainerContributor.kt:31)
    ...
```
See details in the attachment:
[stacktrace.txt](https://github.com/Him188/kotlin-jvm-blocking-bridge/files/5638164/stacktrace.txt)

**Comment by Him188** [comment]
Thank you for your report. Please check your Kotlin version. 
This plugin only supports Kotlin 1.4.20, while the Kotlin plugin bundled in IJ 2020.3 is 1.4.10. You may update in Settings | Language and frameworks | Kotlin. 
If IJ can't find version 1.4.20, switch to Kotlin EAP channel.

**Comment by Samarium150** [comment]
Solved by switching to EAP 1.4.x channel which installed 1.4.20. Thanks.
